third_party/move/move-compiler-v2/tests/testsuite.rs

```diff
@@ -0,0 +1,63 @@
+// Copyright © Aptos Foundation
+// Parts of the project are originally copyright © Meta Platforms, Inc.
```

Comment:
I tend to disagree because this is just a generic driver which is used in this or another form in multiple places of code based on the Move model. This is boilerplate to get things running. The same applies for the testsuite. This is not the same or copied code. Its just a starting point. In the next PR, it will look already rather different. Stay tuned.